### DISCUSSION TITLE
FLoC?

### DISCUSSION THREAD

**Discussion by savolla** [bromite/bromite#1058] [Q&A]
will Bromite users be affected by FLoC?

**Comment by chromer030** [comment]
https://amifloced.org/

**Reply by savolla** [reply]
Wow. Thank you!

**Comment by chromer030** [comment]
Ublock now will block FloC : 
https://github.com/uBlockOrigin/uBlock-issues/issues/1553

**Comment by uazo** [comment]
I'll take a look at the code and let you know.

**Reply by csagan5** [reply]
@uazo the feature is disabled by default

**Comment by deepeer** [comment]
Is possible to remove floc for better privacy

Federated Learning of Cohorts
In chrome://components

floc is new google tracking

https://blog.google/products/ads-commerce/2021-01-privacy-sandbox/

https://github.com/WICG/floc

https://www.eff.org/deeplinks/2021/03/googles-floc-terrible-idea
